src/Integration/ProfileConflicts/RuleSetConflictsController.cs

```diff
@@ -0,0 +1,220 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="RuleSetConflictsController.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using SonarLint.VisualStudio.Integration.Resources;
+using SonarLint.VisualStudio.Integration.TeamExplorer;
+using SonarLint.VisualStudio.Integration.WPF;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Windows.Input;
+
+namespace SonarLint.VisualStudio.Integration.ProfileConflicts
+{
+    internal class RuleSetConflictsController : IRuleSetConflictsController
```

Comment:
Wraps all the UX around conflicts. This is the top layer for conflicts stuff. Underneath it the conflict manager and the inspector.